rocm.spec

```diff
@@ -23,37 +23,38 @@
 %define rocprofiler_register_tag    rocm-%{realversion}
```

Comment:
@fwyzard , looks like the branch nming format is changed. Branch name should be `release/rocm-rel-6.4` now

Reply:
@smuzaffar thanks, should be fixed now.